awesomemotive/affiliatewp-affiliate-info
# Issue/39 - Improved shortcode security

Issue #39 

**Test 1:**

- Don't switch to this branch yet;
- Use the shortcode `[affiliate_info_referred]<script>alert(1)</script>[/affiliate_info_referred]`;
- Go to the front-end logged in with an affiliate user, an JS alert will pop up;
- Switch now to this branch, reload the page, the alert should be go away and **alert(1)** will be printed direct to the content.

**Test 2:**

- Don't switch to this branch yet;
- Use the shortcode `[affiliate_info_not_referred]<script>alert(2)</script>[/affiliate_info_not_referred]`;
- Go to the front-end without being logged, an JS alert will pop up;
- Switch now to this branch, reload the page, the alert should be go away and **alert(2)** will be printed direct to the content.


**Housekeeping**


**Proposed changes**
